packages/compose/src/hooks/use-dragging/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useCallback, useState, useRef } from '@wordpress/element';
+
+export default function useDragging( { onDragStart, onDragMove, onDragEnd } ) {
```

Comment:
Can we add some docs for this hook, it's not clear to me how it's supposed to work? Instead of these callbacks, should it receive a ref instead?

Reply:
Hi @youknowriad, I added a readme file as suggested.